async_upnp_client/dlna.py

```diff
@@ -17,18 +16,17 @@
 from xml.sax.handler import ErrorHandler
 
 from urllib.parse import quote_plus
-from urllib.parse import urljoin
 from urllib.parse import urlparse
 from urllib.parse import urlunparse
 
 from defusedxml.sax import parseString
-from didl_lite import didl_lite
+from didl_lite import didl_lite  # type: ignore
```

Comment:
Just a note I guess, now that didl_lite has a bunch of type hints and hopefully will get some more, perhaps this will not be necessary that much longer? It probably still is at the moment though.